app/services/logger.js

```diff
@@ -5,25 +5,33 @@ export default Ember.Service.extend({
   session: Ember.inject.service(),
   rollbar: Ember.inject.service(),
 
+  raiseError: function(errorData, reason, currentUser) {
```

Comment:
You can directly use `raiseError(arguments)`.. No need use function keyword

Reply:
yeah sure @Darshan-Chauhan.